### PR TITLE
Fix readthedoc/API reference

### PR DESCRIPTION
**Pull request by decarlof**
- fixes [this](https://tomocupy.readthedocs.io/en/latest/api/tomocupy.find_center.html) issue 
- adds missing bib references
- clear all readthedocs build warnings

### CODE REVIEW

**Comment by decarlof** [comment]
@nikitinvv can we delete writer_nolinks.py? Is not used anywhere and the h5nolinks option is handled in writer.py

File: docs/source/conf.py [hunk 1]
```diff
@@ -324,6 +324,7 @@
     'os',
     'pathlib',
     'pytorch_wavelets',
```
Comment (nikitinvv) [inline docs/source/conf.py]: Could you remove pytorch_wavelets and torch from dependencies (and maybe synchronize the deps from setup.py file)? I have rewritten all torch fucntions with cupy and now we dont need these dependencies.

**Review by nikitinvv — CHANGES_REQUESTED** [review]

**Comment by nikitinvv** [comment]
> @nikitinvv can we delete writer_nolinks.py? Is not used anywhere and the h5nolinks option is handled in writer.py

Yes, sure

**Comment by decarlof** [comment]
@nikitinvv done and ready for you to merge

**Comment by nikitinvv** [comment]
thanks @decarlof

**Pull request merged**